pkg/apis/rollouts/v1alpha1/types.go

```diff
@@ -80,6 +80,10 @@ type BlueGreenStrategy struct {
 	// resumed the new replicaset will be full scaled up before the switch occurs
 	// +optional
 	PreviewReplicaCount *int32 `json:"previewReplicaCount,omitempty"`
+	// AutoPromoteEnabled indicates if the rollout should automatically promote the new ReplicaSet
+	// to the active service or enter a paused state. If not specified, the default value is true.
+	// +optional
+	AutoPromoteEnabled *bool `json:"autoPromoteEnabled,omitempty"`
```

Comment:
We are inconsistent: `AutoPromoteEnabled` vs. `AutoPromotionSeconds`. To be consistent it should be: `AutoPromotionEnabled`

Reply:
fixed